### PR TITLE
[Keyboard] Add support E8ghty

### PR DESCRIPTION
**Pull request by trnthsn**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/trnthsn/e8ghty/info.json [hunk 1]
```diff
@@ -0,0 +1,585 @@
...
+    },
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_all"
+    },
+    "community_layouts": [
+        "tkl_f13_ansi_tsangan_split_bs_rshift",
+        "tkl_f13_ansi_tsangan",
+        "tkl_ansi_tsangan",
+        "tkl_ansi_tsangan_split_bs_rshift"
+    ],
```
Comment (dunk2k) [inline keyboards/trnthsn/e8ghty/info.json:54]: 1. remove `layout_aliases`; as it's not used in keymap(s)
2. tidy community layouts literal array
```suggestion
    "community_layouts": ["tkl_ansi_tsangan", "tkl_ansi_tsangan_split_bs_rshift", "tkl_f13_ansi_tsangan", "tkl_f13_ansi_tsangan_split_bs_rshift"],
```

File: keyboards/trnthsn/e8ghty/config.h [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define WS2812_SPI_DRIVER SPID2
+#define WS2812_SPI_MOSI_PAL_MODE 0
+#define WS2812_SPI_SCK_PAL_MODE 0
+#define WS2812_SPI_SCK_PIN B13
```
Comment (dunk2k) [inline keyboards/trnthsn/e8ghty/config.h:23]: EoF tidy
```suggestion
#define WS2812_SPI_SCK_PIN B13

```

File: keyboards/trnthsn/e8ghty/halconf.h [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define HAL_USE_SPI TRUE
+
+#include_next <halconf.h>
+
```
Comment (dunk2k) [inline keyboards/trnthsn/e8ghty/halconf.h:22]: EoF tidy
```suggestion
```

File: keyboards/trnthsn/e8ghty/info.json [hunk 4]
```diff
@@ -0,0 +1,585 @@
...
+                {"label": "Fn", "matrix": [5, 12], "x": 12.5, "y": 5.25},
+                {"label": "Ctrl", "matrix": [5, 13], "x": 13.5, "y": 5.25, "w": 1.5},
+
+                {"label": "\u2190", "matrix": [5, 14], "x": 15.25, "y": 5.25},
+                {"label": "\u2193", "matrix": [5, 15], "x": 16.25, "y": 5.25},
+                {"label": "\u2192", "matrix": [5, 16], "x": 17.25, "y": 5.25}
+            ]
+        }
+    }
+}
```
Comment (dunk2k) [inline keyboards/trnthsn/e8ghty/info.json:585]: EoF tidy
```suggestion
}

```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**